wazo_webhookd/bus.py

```diff
@@ -155,3 +158,13 @@ def _create_bindings(self, event_names, user_uuid, wazo_uuid):
             result.append(binding)
 
         return result
+
+    @staticmethod
+    def no_error_callback(callback, body, message):
```

Comment:
Not a standard, but using same naming than other services would simplify comprehension (agentd, calld, chatd, confd, phoned)
```suggestion
    def _on_bus_message(callback, body, message):
```

Reply:
Oh I see the origin (dird) ... maybe it's time to make standard :trollface:

Reply:
Standard is good :) Thanks for noticing!